js/temperature_humidity_sensor.js

```diff
@@ -0,0 +1,53 @@
+export class TemperatureHumiditySensor {
+    constructor(data) {
+        this.data = data
+    }
+
+    data() {
+        return this.data
+    }
+
+    get inside() {
+        const inside_data = this.data.find(r => r.name == "Music room")
```

Comment:
At some point we may want this to be configurable, but this works for now

Reply:
Agree. I think it would be straightforward to add that later.